extensions/AtlasMongoDb/AtlasMongoDb/ConfigurationHelper.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.KernelMemory;
+using Microsoft.KernelMemory.ContentStorage;
+using Microsoft.KernelMemory.MemoryStorage;
+
+namespace Alkampfer.KernelMemory.AtlasMongoDb;
+
+public static partial class KernelMemoryBuilderExtensions
+{
+    /// <summary>
+    /// Adds Mongodb as a storage service
+    /// </summary>
+    /// <param name="builder">The kernel builder</param>
+    /// <param name="configuration">Configuration for Mongodb</param>
+    public static IKernelMemoryBuilder WithAtlasMemoryDb(
```

Comment:
as a user I might want to use MongoDb Atlas only for memory records, keeping files on disk. Could you add more extension methods to allow that?

Reply:
I would also rename the method, similar to other places. E.g. here "Atlas" I think should be "MongoDbAtlas"

Reply:
Done